app/code/Magento/Customer/Model/Backend/Search/Customer.php

```diff
@@ -3,32 +3,19 @@
  * Copyright © Magento, Inc. All rights reserved.
```

Comment:
@ishakhsuvarov How about if I deprecate the class, leave it empty and make it extend my new class (the one in the right module)? Would that solve the issue for 3rd party code that may use plugins on it?